pkg/arvo/sys/lull.hoon

```diff
@@ -1373,6 +1378,9 @@
         ::    not allowed.
         ::
         [%bound accepted=? =binding]
+        ::  notification that a cache entry has changed
+        ::
+        [%grow =path]
```

Comment:
I think we should change `%grow` to `%grew` for this gift, and probably move it to the generic `$gift-arvo` type and then `$>` that here.  Gall will likely use this gift too, and we're adding a `%grow` note to agents so that they can grow their own paths.  `%grew` matches nicely with `%grow`, while implying it's a notification that the path was just grown.

Reply:
That can be done after merge, to be clear.